console/forms.py

```diff
@@ -0,0 +1,8 @@
+from flask_wtf import Form
+from wtforms import StringField, PasswordField
+from wtforms.validators import DataRequired
+
+
+class NewUserForm(Form):
+    email = StringField('email', validators=[DataRequired()])
+    password = PasswordField('password', validators=[DataRequired()])
```

Comment:
Are there any IA password requirements we need to set here?

Reply:
I wouldn't go crazy (very much against too prescriptive password rules as they just make things less secure) but a minimum length wouldn't go amiss.